RevenueCatUI/Templates/LinTemplate5Step1View.swift

```diff
@@ -0,0 +1,182 @@
+//
+//  File.swift
+//  
+//
+//  Created by Guillaume LAURES on 28/06/2024.
+//
+
+import Foundation
+import RevenueCat
+import SwiftUI
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+struct LinTemplate5Step1View: TemplateViewType {
+    static var paywallFirstStepImageBackgroundColor = Color(red: 255 / 255.0, green: 239 / 255.0, blue: 94 / 255.0)
+    static var bundle = Foundation.Bundle.module
+    let configuration: TemplateViewConfiguration
+    @Environment(\.userInterfaceIdiom)
+    var userInterfaceIdiom
+    @Environment(\.verticalSizeClass)
+    var verticalSizeClass
+    @Environment(\.horizontalSizeClass)
+    var horizontalSizeClass
+    @EnvironmentObject
+    private var introEligibilityViewModel: IntroEligibilityViewModel
+    @State
+    private var selectedPackage: TemplateViewConfiguration.Package
+    
+    var isEligibleToFreeTrial: Bool {
```

Comment:
IDK why, but RC names it with 'eligibleFor', not To

Reply:
Yeah I guess I can revert this. I think RC is remaining very generic because the offer can be something else than free time.

Reply:
I changed this slightly in the next branch (the one for navigation) so I propose to leave it like that on this one and fix/check on the next one.